x/auth/keeper/keeper.go

```diff
@@ -145,9 +160,17 @@ func NewAccountKeeper(
 		panic(err)
 	}
 	ak.Schema = schema
+
+	for _, opt := range opts {
+		opt(&ak)
+	}
 	return ak
 }
 
+func (ak AccountKeeper) IsUnorderedTransactionsEnabled() bool {
```

Comment:
```suggestion
func (ak AccountKeeper) UnorderedTransactionsEnabled() bool {
```
`Is` sounds ungrammatical

Reply:
fixed [rename IsUnorderedTransactionsEnabled](https://github.com/cosmos/cosmos-sdk/pull/24573/commits/925cb67c98f8440317a8b1c6f7db7cc3bcd77a16)